handlers/hormuud/handler_test.go

```diff
@@ -201,16 +198,8 @@ var tokenTestCases = []OutgoingTestCase{
 					"grant_type": {"password"},
 				},
 			},
-			{
-				Headers: map[string]string{
-					"Content-Type":  "application/json",
-					"Accept":        "application/json",
-					"Authorization": "Bearer ",
-				},
-				Body: `{"mobile":"250788383383","message":"Simple Message","senderid":"2020","mType":-1,"eType":-1,"UDH":""}`,
-			},
 		},
-		ExpectedError: courier.ErrResponseStatus,
```

Comment:
?

Reply:
We are not reaching to the place we call the send mock since the token fails